cli/lib/kontena/cli/stacks/logs_command.rb

```diff
@@ -9,13 +9,20 @@ class LogsCommand < Kontena::Command
     banner "Shows logs from services in a stack"
 
     parameter "NAME", "Stack name"
+    parameter "[SERVICE] ...", "Service names"
```

Comment:
I would suggest `--service` for consistency with `kontena grid logs`, and keep the additional arguments for possible multi-stack log tailing (`kontena stack logs stack-1 stack-2`)?